src/include/rados/librados.h

```diff
@@ -406,7 +406,8 @@ CEPH_RADOS_API int rados_create2(rados_t *pcluster,
  * @returns 0 on success, negative error code on failure
  */
 CEPH_RADOS_API int rados_create_with_context(rados_t *cluster,
-                                             rados_config_t cct);
+                                             rados_config_t cct,
+                                             const char *name);
```

Comment:
we can't change the ABI here; need to define a new variant of this function to add the arg.